server/otlp/exporter.go

```diff
@@ -3,49 +3,33 @@ package otlp
 import (
 	"context"
 	"fmt"
-	"net"
 	"strings"
 
 	"github.com/kubeshop/tracetest/server/model"
 	"github.com/kubeshop/tracetest/server/traces"
 	"go.opentelemetry.io/otel/trace"
 	pb "go.opentelemetry.io/proto/otlp/collector/trace/v1"
 	v1 "go.opentelemetry.io/proto/otlp/trace/v1"
-	"google.golang.org/grpc"
 )
 
-type Server struct {
-	pb.UnimplementedTraceServiceServer
-
-	addr string
-	db   model.Repository
-
-	gServer *grpc.Server
+type IExporter interface {
+	Export(ctx context.Context, request *pb.ExportTraceServiceRequest) (*pb.ExportTraceServiceResponse, error)
```

Comment:
I know the OTLP proto calls it an `export` operation, but in the Tracetest perspective, it should be Import or Ingest (I prefer this one). Can you change it to make it more clear of what this interface should be doing?